src/tests/ftest/util/command_utils.py

```diff
@@ -149,17 +159,22 @@ def _run_process(self):
         # Clear any previous run results
         self.result = None
         command = str(self)
+        if raise_exception is not None:
+            do_raise = raise_exception
+        else:
+            do_raise = self.exit_status_exception
+
         try:
             # Block until the command is complete or times out
             self.result = run_command(
-                command, self.timeout, self.verbose, self.exit_status_exception,
+                command, self.timeout, self.verbose, do_raise,
```

Comment:
Then, to follow on from the previous comment:
```suggestion
                command, self.timeout, self.verbose, raise_exception,
```